docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+services:
+  pythonapp:
+    image: todoapp:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: todoapp
+    ports:
+      - "8080:8080"
+    environment:
+      - ENV=local
```

Comment:
What is the purpose of this variable? Because it doesn't have any references in app

Reply:
No purpose. Deleted it. I don't know why but there was no comment "No need to set up CMD and ENTRYPOINT, it is set up in base image" before.  Now I removed it.

Reply:
It's ok. GitHub review comments work a few curiously and they are not always processed immediately